upsilonconf/io.py

```diff
@@ -225,26 +225,109 @@ def _get_io_function(path: Path, write: bool = False):
     raise ValueError(f"unknown config file extension: '{ext}'")
 
 
-def load(path: Union[Path, str]) -> Configuration:
+def __replace_in_keys(
```

Comment:
I probably don't mention this anywhere (yet), but I use `black` for formatting my code. Would it be possible to run [`black`](https://black.readthedocs.io/en/stable/getting_started.html) so that formatting remains consistent?

Reply:
I only use `pep8`, which does not have such strong views as `black`. If you prefer `black`, I can certainly check with it!

Reply:
Done in https://github.com/hoedt/upsilonconf/pull/3/commits/4f5d2aaa67563d10771770d6d6b0c8ddce9a11ca.